src/app/personside/infotabs/saksoversikt/DokumentKomponent.tsx

```diff
@@ -108,7 +107,9 @@ function utgåendeTekst(mottaker: Entitet, mottakernavn: string) {
 }
 
 function formaterDatoOgAvsender(brukernavn: string, dokument: DokumentMetadata) {
-    const dato = moment(saksdatoSomDate(dokument.dato)).format('DD.MM.YYYY');
+    const saksdato = dokument.dato;
+    const dato = formaterDato(`${saksdato.år}.${saksdato.måned}.${saksdato.dag}`);
+    console.log(dato, dokument.dato);
```

Comment:
Dette er undersøkelser for å finne ut av den +1måned-buggen?

Reply:
Ja, det løser den buggen.

Reply:
fjerne console.log da 🗡